src/test/com/yetanalytics/lrs/xapi/activities_test.cljc

```diff
@@ -5,8 +5,66 @@
    [com.yetanalytics.test-support :refer [failures stc-opts]]
    [com.yetanalytics.lrs.xapi.activities :as ac]))
 
+(def act-id-only
+  {"id" "http://www.example.com/tincan/activities/multipart"})
+
+(def act-full
+  {"id"         "http://www.example.com/tincan/activities/multipart"
+   "objectType" "Activity"
+   "definition" {"type"        "http://www.example.com/activity-types/test"
+                 "name"        {"en-US" "Multi Part Activity"
+                                "zh-CN" "多元部分Activity"}
+                 "description" {"en-US" "Multi Part Activity Description"
+                                "zh-CN" "多元部分Activity的简述"}}})
+
+(def act-en
+  {"id"         "http://www.example.com/tincan/activities/multipart"
+   "objectType" "Activity"
+   "definition" {"type"        "http://www.example.com/activity-types/test"
+                 "name"        {"en-US" "Multi Part Activity"}
+                 "description" {"en-US" "Multi Part Activity Description"}}})
+
+(def act-zh
+  {"id"         "http://www.example.com/tincan/activities/multipart"
+   "objectType" "Activity"
+   "definition" {"type"        "http://www.example.com/activity-types/test"
+                 "name"        {"zh-CN" "多元部分Activity"}
+                 "description" {"zh-CN" "多元部分Activity的简述"}}})
+
+(def act-matching
+  {"id" "aaa://aaa.aaa.aaa/aaa"
+   "objectType" "Activity"
+   "definition"
+   {"interactionType" "matching"
+    "source"          [{"id" "0"}]
+    "target"          [{"id" "A"}]}})
+
+(def act-numeric
+  {"id" "aaa://aaa.aaa.aaa/aaa"
+   "objectType" "Activity"
+   "definition"
+   {"interactionType" "numeric"}})
+
 (deftest merge-activity-test
-  (testing "merging activities"
+  (testing "merging activities with differing detail levels"
```

Comment:
Actually I should suggest to add a test case for activities with extensions. I'm 90% sure the code will handle it correctly (hence my approval stands), but given that extensions are always finicky it wouldn't hurt to add.

Reply:
We don't want to merge extension map values since we don't know their semantics. Merging the extensions map itself is fine tho.